afids_utils/ext/json.py

```diff
@@ -0,0 +1,154 @@
+"""Methods for handling .json files associated with AFIDS"""
+from __future__ import annotations
+
+import json
+from importlib import resources
+from os import PathLike
+
+from afids_utils.afids import AfidPosition, AfidSet
+from afids_utils.exceptions import InvalidFileError
+
+
+class ControlPoint:
+    id: str
+    label: str
+    description: str
+    associatedNodeID: str
+    position: list[float]
+    orientation: list[float]
+    selected: bool
+    locked: bool
+    visibility: bool
+    positionStatus: str
+
+
+def _get_metadata(
+    in_json: dict[str, bool | float | str | list[float]]
+) -> tuple[str, str]:
+    """Internal function to extract metadata from json files
+
+    Note: Slicer version is not currently included in the json file
+
+    Parameters:
+    -----------
+    in_json
+        Data from provided json file to parse metadata from
+
+    Returns
+    -------
+    parsed_version
+        Slicer version associated with fiducial file
+
+    parsed_coord
+        Coordinate system of fiducials
+
+    Raises
+    ------
+    InvalidFileError
+        If header is invalid from .json file
+    """
+
+    # Update if future json versions include Slicer version
+    parsed_version = "Unknown"
+    parsed_coord = in_json["coordinateSystem"]
```

Comment:
I would just set `in_json["coordinateSystem"]` directly as the parameter to this function

Reply:
I think there is an argument to be made for the metadata that we don't even need a separate `_get_metadata` function for the jsons, but can change that if (when) we drop the slicer versioning.